### PR TITLE
Testnet 2 genesis

### PR DESCRIPTION
**Pull request by askolesov**

### CODE REVIEW

File: persistent_chains/testnet-2/genesis.json [hunk 1]
```diff
@@ -0,0 +1,197 @@
+{
+  "genesis_time": "2021-09-24T14:47:05.733203Z",
+  "chain_id": "cheqd-testnet-2",
```
Comment (ankurdotb) [inline persistent_chains/testnet-2/genesis.json:3]: Can we just call this cheqd-testnet?

File: persistent_chains/testnet-2/genesis.json [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+      "balances": [],
+      "supply": [],
+      "denom_metadata": []
+    },
+    "capability": {
+      "index": "1",
+      "owners": []
+    },
+    "cheqd": {
+      "nymList": []
```
Comment (ankurdotb) [inline persistent_chains/testnet-2/genesis.json:52]: Should this still be called 'nymList` when NYMs are being removed? What is this for anyway?

**Review by ankurdotb — CHANGES_REQUESTED** [review]
Minor comments to check before we finalise

File: persistent_chains/testnet-2/genesis.json (hunk 2 above)
Comment (askolesov) [reply persistent_chains/testnet-2/genesis.json:70]: Discussed in slack: decided to go with `null` and make genesis validation to accept it

File: persistent_chains/testnet-2/genesis.json (hunk 1 above)
Comment (askolesov) [reply persistent_chains/testnet-2/genesis.json:3]: Discussed in slack: decided to go with `cheqd-testnet-2`

**Pull request merged**